### DISCUSSION TITLE
Snowflake not working on Calcul Québec server

### DISCUSSION THREAD

**Discussion by RechercheStochastique** [anyonlabs/Snowflake.jl#28] [Q&A] [answered]
I have successfully add the Snowflake pakage using
julia> Pkg.add("Snowflake")

Then I try the example and get an error:
julia> using Snowflake

julia> c = QuantumCircuit(qubit_count=2, bit_count=0)
ERROR: UndefVarError: QuantumCircuit not defined
Stacktrace:
 [1] top-level scope

Any idea?

**Comment by npoirie1** [comment, marked as answer]
Hello Mr. Gagnon,
While Snowflake is in its alpha phase, you should use the main branch of the GitHub repository. You can do so by calling

julia> Pkg.add(url="https://github.com/anyonlabs/Snowflake.jl", rev="main")

instead of

julia> Pkg.add("Snowflake")

**Reply by RechercheStochastique** [reply]
Thanks. That's what I did. Seems ok.
